### PR TITLE
Reorder documentation index

### LINKED ISSUE #1914: Reorder documentation
I am thinking of reordering the sections of the documentation into a more logical order. And maybe add cross headings.
I have something like this in mind. What do you think of that?

#### Getting Started & Configuration

- General User Information
- Getting Started with LinuxCNC
- Configuration Wizards
- Configuration
- HAL (Hardware Abstraction Layer)
- Hardware Drivers
- Hardware Examples
- Classicladder
- Advanced Topics
- Integrator Information

#### Usage

- User Interfaces
- G-code Programming

#### Customization & Development

- Virtual Control Panels
- User Interface Programming
- Developer Information
- Glossary, Copyright, History & Overview

### PR DESCRIPTION
**Pull request by hansu**
Like mentioned in #1914.
Now also adapted the PDF version.
See http://buildbot.linuxcnc.org/doc/scratch/v2.9.0~pre0~1914-reorder-documentation~da5be815b/ for a preview.

Closes #1914

### CODE REVIEW

File: docs/src/index.tmpl [hunk 1]
```diff
@@ -88,17 +88,23 @@ function setup_page(){
   <img src="linuxcnc-logo-chips.png" alt="LinuxCNC Logo" width="175"/>
 </div>
 
-<h3>LinuxCNC version <strong>@VERSION@</strong></h3>
+<!-- <h3 align="center">LinuxCNC version <strong>@VERSION@</strong></h3>
+<h2>Documentation</h2> -->
```
Comment (petterreinholdtsen) [inline docs/src/index.tmpl:92]: Why is this part commented out?

Comment (hansu) [reply docs/src/index.tmpl:92]: Good catch, these lines should actually be removed.

**Comment by hansu** [comment]
So I guess I'll merge it if nobody complains about it? :D

**Comment by andypugh** [comment]
OK with me.

**Pull request merged**